apps/app/src/pages/Repository/Settings.js

```diff
@@ -9,55 +10,117 @@ import {
 import { useRepository, useToggleRepository } from "./RepositoryContext";
+import { Box } from "@xstyled/styled-components";
+import { useMutation } from "@apollo/client";
+
+const UPDATE_BASELINE_BRANCH = gql`
+  mutation UpdateBaselineBranch($repositoryId: String!, $branchName: String!) {
+    updateBaselineBranch(repositoryId: $repositoryId, branchName: $branchName) {
+      id
+      baselineBranch
+    }
+  }
+`;
 
 export function RepositorySettings() {
   const repository = useRepository();
+  const { enabled } = repository;
   const { toggleRepository, loading, error } = useToggleRepository();
-  const { owner, enabled } = repository;
+
+  const [baselineBranch, setBaselineBranch] = useState(
+    repository.baselineBranch
+  );
+
+  const [
+    updateBaselineBranch,
+    { loading: baseBranchMutationLoading, error: baseBranchMutationError },
+  ] = useMutation(UPDATE_BASELINE_BRANCH);
+
   return (
     <Container>
       <Helmet>
         <title>Settings</title>
       </Helmet>
       <Boxer my={4}>
         {enabled && (
-          <Card>
-            <CardHeader>
-              <CardTitle>Environment Variables</CardTitle>
-            </CardHeader>
-            <CardBody>
-              <CardText>
-                To send data to Argos CI you will need to configure a{" "}
-                <FadeLink
-                  href="https://github.com/argos-ci/argos-cli"
-                  target="_blank"
-                  color="darker"
-                >
-                  CLI
-                </FadeLink>{" "}
-                with a client key (usually referred to as the ARGOS_TOKEN
-                value).
-                <br />
-                ARGOS_TOKEN is a project-specific, it should be kept secret.
-                <br />
-                For more information on integrating Argos CI with your
-                application take a look at our{" "}
-                <FadeLink
-                  color="darker"
-                  href={`/${owner.login}/${repository.name}/getting-started`}
-                  variant="primary"
-                >
-                  documentation.
-                </FadeLink>
-              </CardText>
-              <Code>ARGOS_TOKEN={repository.token}</Code>
-            </CardBody>
-          </Card>
+          <>
+            <Card>
+              <CardHeader>
+                <CardTitle>Argos Token</CardTitle>
+              </CardHeader>
+              <CardBody>
+                <CardText fontSize={16}>
+                  Use this <InlineCode>ARGOS_TOKEN</InlineCode> to authenticate
+                  your repository when you send screenshots to Argos.
+                </CardText>
+                <Alert variant="warning">
+                  This token should be kept secret.
+                </Alert>
+                <Code>ARGOS_TOKEN={repository.token}</Code>
+                <CardText fontSize={16}>
+                  Look at our documentation for more information about Argos{" "}
```

Comment:
`Read our documentation for more information about installing Argos and using it.` ?